logdna/client_test.go

```diff
@@ -0,0 +1,91 @@
+package logdna
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClient_View(t *testing.T) {
+	body := make(map[string](interface{}))
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		json.NewDecoder(r.Body).Decode(&body)
```

Comment:
this is where you'd want to respond from the server to mimic our API. right now it's not doing anything.

you should also be able to make assertions on the request itself inside of the handler here (e.g. are the correct fields present with the expected values, etc)